### PR TITLE
return non-zero exit code for any failed tests

### PR DESCRIPTION
**Pull request by smoors**

### CODE REVIEW

File: run_weekly.sh [hunk 1]
```diff
@@ -2,18 +2,27 @@
...
+    "./run.sh -c cp2k_tests -n CP2KTestSingleNode"
+)
 
+for cmd in "${cmds[@]}"; do
+    eval "$cmd"
+    exitcode=$?
+    ((total+=exitcode))
+done
+
+exit $total
```
Comment (wpoely86) [inline run_weekly.sh]:
```suggestion
exit ${total:-0}
```

File: run_weekly.sh [hunk 2]
```diff
@@ -2,18 +2,27 @@
...
+    "./run.sh -c blas-tester"
+    "./run.sh -c gromacs_bench -n GMXBenchMEMSingleNode"
+    "./run.sh -c gromacs_bench -n GMXBenchMEMSingleNodeGPU"
+    "./run.sh -c cp2k_tests -n CP2KTestSingleNode"
+)
 
+for cmd in "${cmds[@]}"; do
+    eval "$cmd"
+    exitcode=$?
+    ((total+=exitcode))
```
Comment (wpoely86) [inline run_weekly.sh:27]: total is not defined or initialized to zero?

File: run_weekly.sh [hunk 3]
```diff
@@ -2,18 +2,27 @@
...
-./run.sh -c cp2k_tests -n CP2KTestSingleNode
+    # single-node tests
+    "./run.sh -c blas-tester"
+    "./run.sh -c gromacs_bench -n GMXBenchMEMSingleNode"
+    "./run.sh -c gromacs_bench -n GMXBenchMEMSingleNodeGPU"
+    "./run.sh -c cp2k_tests -n CP2KTestSingleNode"
+)
 
+for cmd in "${cmds[@]}"; do
+    eval "$cmd"
```
Comment (wpoely86) [inline run_weekly.sh:25]: might it be convenient to do an echo first? Might make debugging easier?

File: run_weekly.sh (hunk 2 above)
Comment (smoors) [reply run_weekly.sh:27]: in my test it is not needed to initialize it, but it's indeed cleaner to do it anyway

**Review by wpoely86 — APPROVED** [review]

**Pull request merged**